cve_bin_tool/cli.py

```diff
@@ -51,6 +49,11 @@
 from cve_bin_tool.version import VERSION
 from cve_bin_tool.version_scanner import VersionScanner
 
+if sys.version_info >= (3, 8):
+    from importlib import metadata as importlib_metadata
+else:
+    import importlib_metadata
```

Comment:
Just checking this is not enough, we will actually need to include `importlib_metadata` dependency for Python less than 3.8 (which is only 3.7 at this point).

You will need to add this line to `requirements.txt`:
```
importlib_metadata;python_version<"3.8"
```

You may ask: but why do all tests pass on Python 3.7 in CI? That's because this package is required by some doc package that we use (`sphinx-markdown-tables` requires `Markdown` which in turn requires our `importlib_metadata`) but the end user might not use docs functionality. And it's better to be explicit about it. Especially because we will most likely remove this workaround at the end of the year when Python 3.7 goes out of support.

Reply:
Oh okay, I was actually just going to ask about the 3.7 tests, thanks for the explanation.
Will add importlib to the requirements file as well as rename the commit to adhere to standards.